torch/csrc/jit/graph_executor.cpp

```diff
@@ -645,6 +646,10 @@ struct GraphExecutorImpl {
     }
     // decomposition pass, decompose certain ops that will be used in the following
     // passes (like batchmm and jit fusion)
+    DecomposeOps(graph);
+    // Autodiff and decompositon pass will replace some parts of graph with new graph
+    // these new graphs usually miss shape information on nodes, so we propagate shapes
+    // PropagateInputShapes(graph);
```

Comment:
why is this commented out?

Reply:
some issues on `nn.Dropout` when enable this, I will need to look into that and will bring the pass back.